docs/pages/setup/operations/scaling.mdx

```diff
@@ -36,20 +36,20 @@ icon="building"
 
 - Teleport v(=teleport.version=) Open Source or Enterprise.
 
-
 ## Hardware recommendations
 
 Set up Teleport with a [High Availability configuration](../reference/backends.mdx).
 
 | Scenario | Max Recommended Count | Proxy | Auth Server | AWS Instance Types |
 | - | - | - | - | - |
 | Teleport Nodes connected to Auth Service | 10,000 | 2x  4 vCPUs, 8GB RAM | 2x 8 vCPUs, 16GB RAM | m4.2xlarge |
+| Teleport Nodes connected to Auth Service | 50,000 | 2x  4 vCPUs, 16GB RAM | 2x 8 vCPUs, 16GB RAM | m4.2xlarge |
```

Comment:
Do these guidelines apply to all `teleport` instances proxying resources in the cluster, e.g., the Kubernetes Service and Database Service?

Reply:
@ptgott Updated, can you take another look?